test_conformance/api/test_kernels.cpp

```diff
@@ -704,3 +722,65 @@ REGISTER_TEST(negative_set_immutable_memory_to_writeable_kernel_arg)
 
     return TEST_PASS;
 }
+
+REGISTER_TEST(negative_set_read_write_image_arg)
+{
+    cl_int error = CL_SUCCESS;
+    clProgramWrapper program;
+    clKernelWrapper kernels[2];
+    clMemWrapper images[2], buffer;
+    const char *test_kernels[2] = { sample_read_only_image_test_kernel,
```

Comment:
We never loop over these arrays, so can we use more descriptive names instead?  I'm thinking of things like:

write_image_kernel, read_image_kernel, write_only_image, read_only_image

Reply:
Done